tests/testconf.json

```diff
@@ -0,0 +1,17 @@
+{
+    "topic": "test-topic",
+    "bootstrap.servers": "$MY_BOOTSTRAP_SERVER_ENV",
+    "schema.registry.url": "$MY_SCHEMA_REGISTRY_URL_ENV",
+    "avro-https": {
+            "schema.registry.url": "$MY_SCHEMA_REGISTRY_SSL_URL_ENV",
+            "schema.registry.ssl.ca.location": "$MY_SCHEMA_REGISTRY_SSL_CA_LOCATION_ENV",
+            "schema.registry.ssl.certificate.location": "$MY_SCHEMA_REGISTRY_SSL_CERTIFICATE_LOCATION_ENV",
+            "schema.registry.ssl.key.location": "$MY_SCHEMA_REGISTRY_SSL_KEY_LOCATION_ENV"
+    },
+    "avro-basic-auth": {
+        "schema.registry.url": "http://localhost:8083",
```

Comment:
I don't think we should provide default hosts/ports in the configuration file (especially when no longer an example), since people could be hitting a service unrelated to this test if unlucky.

Commenting them out is fine.

Reply:
Since everything is dockerized this is largely irrelevant no?